pkg/kubelet/cri/remote/remote_runtime.go

```diff
@@ -796,5 +797,25 @@ func (r *remoteRuntimeService) CheckpointContainer(options *runtimeapi.Checkpoin
 }
 
 func (r *remoteRuntimeService) GetContainerEvents(containerEventsCh chan *runtimeapi.ContainerEventResponse) error {
-	return nil
+	containerEventsStreamingClient, err := r.runtimeClient.GetContainerEvents(context.Background(), &runtimeapi.GetEventsRequest{})
+
+	if err != nil {
+		klog.ErrorS(err, "GetContainerEvents failed to get streaming client")
+		return err
+	}
+
+	for {
+		resp, err := containerEventsStreamingClient.Recv()
+		if err == io.EOF {
+			return err
+		}
+		if err != nil {
+			klog.ErrorS(err, "cannot receive container event")
```

Comment:
failed to receive streaming container event

Reply:
Updated!